### PR TITLE
Feature/count cards

### PR DESCRIPTION
**Pull request by RobertWheel**
counts cards and knows when the game is done

### CODE REVIEW

File: source/CardGames.py [hunk 1]
```diff
@@ -85,6 +85,17 @@ def getCard( suit, value):
       return card
   return None
 
+####################################################################################
+def count_cards(deck):
```
Comment (andredavii) [inline source/CardGames.py]: Your function looks good. However, it seems like there is room left for error handling. Is there any possibility the function could be imputed with something other than a valid deck object?

Comment (RobertWheel) [reply source/CardGames.py]: well it only calls deck, so there shouldent beb any issue of anything else being pushed through 👍

Comment (andredavii) [reply source/CardGames.py]: Got it. Great work!

File: test/tutorial.py [hunk 2]
```diff
@@ -6,9 +6,25 @@
...
+        break
+        #if N then ask them to re enter their name
+    if confirm =="N":
+        print("You have selected 'NO', please select another name....")
+        name = input("Please re-enter your desired name: ")
+        #if its anything else besides a Y or N make them re type their responce
+    else:
+        print("Looks like there was a typo in your responce, please try again and remember to use Capital letters.")
+
+players= [Player(name)]
```
Comment (psk44) [inline test/tutorial.py]: This test seems to operate well! It is simple and does what it is intended to do. If you want to improve it further, you can add more error handling to handle unexpected inputs from the user.

Comment (RobertWheel) [reply test/tutorial.py]: Thanks! It takes all kinds of responses and only looks for 2 responses, and if it isn't those responses, then it won't crash but prompt them to try again 👍

**Pull request merged**